### PR TITLE
234 professor attach books

### PR DESCRIPTION
**Pull request by oceancypress**
test: log in as prof 
username rexifer password rexifer1
or any prof really
select a section row 
see references if any
add a reference

### CODE REVIEW

File: student-info-system/professor/views.py [hunk 1]
```diff
@@ -47,3 +58,37 @@ def student(request, studentid):
     ssects = stud.sectionstudent_set.all()
     data = {'student': stud, 'ssects': ssects}
     return render(request, 'professor/student.html', data)
+
+
+@role_login_required(Profile.ACCESS_PROFESSOR)
+def add_reference(request, sectionid):
+    data = {'sectionid': sectionid}
```
Comment (bjmckenz) [inline student-info-system/professor/views.py:65]: why do you go through section first? Why not index on courses in major?

**Review by bjmckenz — APPROVED** [review]
How come this PR includes the changes from other stuff? did you "pre-merge" the messages things?

File: student-info-system/professor/views.py (hunk 1 above)
Comment (oceancypress) [reply student-info-system/professor/views.py:65]: wrote this before I knew it was by course

**Comment by jeffreicher** [comment]
Can we make the links clickable so they open a new tab on click? If it's a difficult addition we can create another story. @oceancypress

**Pull request merged**